with-gatsby/babel.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = function(api) {
   api.cache(true);
   return {
-    presets: ['babel-preset-expo'],
+    presets: ["babel-preset-expo"],
+    plugins: [["babel-plugin-remove-graphql-queries",{},"test-name"]]
```

Comment:
@EvanBacon , in Gatsby world it's not very common to have a babelrc or babel config at the root of the project.

But for Gatsby it seems required to use the "babel-plugin-remove-graphql-queries" (at least unless I figure out how to merge the Gatsby/Expo js loaders into a single one...)

It would be convenient if you exposed me a way to customize the default babel loader here and add the required gatsby plugin without creating a local babel config.

Basically, I want my Gatsby plugin to add the babel plugin here: https://github.com/expo/expo-cli/blob/f48370005f016f4b8700b31da9f1d51b4ac142a0/packages/webpack-config/src/loaders/createBabelLoader.ts#L139

Reply:
Not the cleanest API, but it is possible to customize the babel loader by finding it and modifying it like so https://github.com/software-mansion/react-native-gesture-handler/blob/master/webpack.config.js

Reply:
yeah will try that. There is likely to be 2 jsloaders and I should just modify one of them.
That's a bit fragile but will probably work

Reply:
that seems to work: https://github.com/slorber/gatsby-plugin-react-native-web/commit/663754ae2748ece5d6be11f19558bc2451a8f4a6